client/nuxt.config.ts

```diff
@@ -80,8 +80,6 @@ export default defineNuxtConfig({
   },
   pinia: {
     autoImports: [
-      'defineStore',
```

Comment:
Why?

Reply:
> [Explicitly import defineStore instead of globally importing it.](https://github.com/Black-Kamelia/Hedera/pull/136/commits/1c1f9ea87d473fca9a644e317f24009e1fc56735) 
[1c1f9ea](https://github.com/Black-Kamelia/Hedera/pull/136/commits/1c1f9ea87d473fca9a644e317f24009e1fc56735)
This is due to a bug in pinia where the defineStore function may be undefined during the initial compilation process.